lib/render/user.php

```diff
@@ -24,14 +24,15 @@ function userAvatar(
     });
 
     if (!$user) {
-        if ($icon !== false && ($icon || !$label)) {
-            return '';
-        }
-
         $userSanitized = $username;
         sanitize_outputs($userSanitized);
 
-        return '<del>' . $userSanitized . '</del>';
+        $iconLabel = '';
+        if ($icon !== false && ($icon || !$label)) {
+            $iconLabel = "<img src='" . media_asset('/UserPic/_User.png') . "' alt='$userSanitized' width='$iconSize' height='$iconSize'>";
+        }
+
+        return "<span class='inline whitespace-nowrap'>$iconLabel <del>$userSanitized</del></span>";
```

Comment:
I don't think you can implicitly put the user name in the text. Suggest putting it in a `title` attribute.
![image](https://user-images.githubusercontent.com/32680403/193368478-e7574602-b98d-4f02-a61d-2c6e7e840994.png)

Reply:
Just noticed 👍